src/essence/Tools/Isochrone/IsochroneTool.js

```diff
@@ -0,0 +1,539 @@
+//New Tool Template
```

Comment:
nit: remove boilerplate comments from the example. either replace them with an explanation of your tool or remove them altogether